manifests/charts/istio-control/istio-discovery/files/grpc-agent.yaml

```diff
@@ -35,17 +67,34 @@ spec:
   {{- else }}
     image: "{{ .ProxyImage }}"
   {{- end }}
+    ports:
+    - containerPort: 15090
+      protocol: TCP
+      name: http-envoy-prom
     args:
     - proxy
     - sidecar
     - --domain
     - $(POD_NAMESPACE).svc.{{ .Values.global.proxy.clusterDomain }}
+    - --proxyLogLevel={{ annotation .ObjectMeta `sidecar.istio.io/logLevel` .Values.global.proxy.logLevel }}
```

Comment:
nit: this was probably intentionally kept out since it does nothing. I don't mind it being here for consistency though

Reply:
Easier to do merges when most lines are identical.